Refs #33476 -- Made management commands use black. (#15412)
django/django: docs/releases/4.1.txt

* The new :option:`migrate --prune` option allows deleting nonexistent
  migrations from the ``django_migrations`` table.

* Python files created by :djadmin:`startproject`, :djadmin:`startapp`,

>   :djadmin:`makemigrations`, and :djadmin:`squashmigrations` will be formatted

Reviewer: ```suggestion
  :djadmin:`makemigrations`, and :djadmin:`squashmigrations` are now formatted
```

Reviewer: Maybe we should add a small section to the [Extra niceties](https://docs.djangoproject.com/en/4.0/ref/django-admin/#extra-niceties) :thinking:
